core/identity-hub-core/src/main/java/org/eclipse/edc/identityhub/core/services/verification/SelfIssuedTokenVerifierImpl.java

```diff
@@ -91,7 +91,7 @@ public Result<List<String>> verify(String token, String participantContextId) {
         var accessTokenString = claimToken.getStringClaim(TOKEN_CLAIM);
         var subClaim = claimToken.getStringClaim(JwtRegisteredClaimNames.SUBJECT);
 
-        TokenValidationRule audMustMatchParticipantIdRule = (at, additional) -> {
+        TokenValidationRule audMustMatchParticipantContextIdRule = (at, additional) -> {
```

Comment:
I would leave this one as `audMustMatchParticipantIdRule` or `audMustMatchParticipantDidRule`